core/frontend/src/components/vehiclesetup/configuration/lights.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="d-flex">
+    <v-card>
+      <v-card-title> Lights 1 (RCIN9) </v-card-title>
+      <v-card-text>
+        Here you can configure what pin outputs the signal for the first set of lights.
+        <v-select
+          v-model="lights1_new_param"
+          :items="recommended_params"
+          :item-text="friendlyName"
+          :item-value="'name'"
+          label="Lights 1"
+        />
+      </v-card-text>
+    </v-card>
+    <v-card>
+      <v-card-title> Lights 2 (RCIN10) </v-card-title>
+      <v-card-text>
+        What pin outputs the signal for the second set of lights.
+        <v-select
+          v-model="lights2_new_param"
+          :items="recommended_params"
+          :item-text="friendlyName"
+          :item-value="'name'"
+          label="Lights 2"
+        />
+      </v-card-text>
+    </v-card>
+    <v-card>
+      <v-card-title> Joystick steps </v-card-title>
+      <v-card-text>
+        How many button presses it takes to go from 0% to 100% brightness.
```

Comment:
I believe that asking the user a percentage per step over the number of buttons press is more friendly. Maybe rephrasing it helps.

Reply:
improved the wording and added a live-updated percentage